CarDealer/Models/CarInventory.cs

```diff
@@ -66,5 +66,11 @@ public void SetOptionsLevel()
         {
             FinalPrice = InitialPrice;
         }
+
+        public CarInventory Clone()
```

Comment:
Clasa CarInventory trebuie sa implementeze interfata ICarPrototype

Reply:
CarInventory si toate obiectele claselor folosite in CarInventory trebuie sa aiba [Serializable] inainte de numele clasei
[Serializable]
class Option ...
Astea sunt Option, Model, Color, Engine si ce obiecte mai sunt acolo in CarInventory